### PR TITLE
update urfave/cli package

### PR DESCRIPTION
**Pull request by hmeng-19**
The new version of urfave/cli supports Int64Flag and Uint64Flag.
	These flag types are useful for setting cpu/mem resources.

Signed-off-by: Haiyan Meng <hmeng@redhat.com>

### CODE REVIEW

**Comment by hmeng-19** [comment]
@mrunalp , PTAL.

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -24,8 +24,8 @@
 		},
 		{
 			"ImportPath": "github.com/urfave/cli",
-			"Comment": "v1.17.0-57-gedb24d0",
-			"Rev": "edb24d02aa3cea2319c33f2836d4a5133907fe4c"
+			"Comment": "v1.18.0-47-g168c954",
+			"Rev": "168c95418e66e019fe17b8f4f5c45aa62ff80e23"
```
Comment (wking) [inline Godeps/Godeps.json]: Can we bump this to v1.18.0 on the nose (urfave/cli@1efa31f)?  It [includes the features you want][1].

[1]: https://github.com/urfave/cli/blob/v1.18.0/CHANGELOG.md#added

Comment (hmeng-19) [inline Godeps/Godeps.json]: @wking , no problem. Fixed it. Thanks. PTAL.

**Comment by mrunalp** [comment]
LGTM

**Comment by wking** [comment]
dc3404ea0 looks good to me.

**Pull request merged**